Added additional information about updating customer's store credit balance (#1520)
magento/merchdocs: src/customers/store-credit-using.md

![]({% link images/images-ee/store-credit-balance.png %}){: .zoom}
_Store Credit Balance_

## Update store credit balance

1. On the _Admin_ sidebar, go to **Customers** > _Operations_ > **All Customers**.

1. Find the customer in the grid.

1. In the _Action_ column, click **Edit**.

1. In the left panel, choose **Store Credit**.

> 1. Choose the website (storefront) for which the balance will be associated.

Reviewer: ```suggestion
1. Choose the website (storefront) to associate with the balance.
```

1. For **Update Balance**, enter the new value.

> 1. To notify customer about balance updating check **Notify Customer by Email** checkbox and choose the store view from **Send Email Notification From the Following Store View**.

Reviewer: ```suggestion
1. To notify the customer about the balance update, select the **Notify Customer by Email** checkbox and choose the store view from **Send Email Notification From the Following Store View**.
```

> 1. Enter the **Comment**

Reviewer: ```suggestion
1. Enter a **Comment** about the change.
```

1. When updates are complete, click <span class="btn">Save and Continue Edit</span> or <span class="btn">Save Customer</span>.

> Updated Balance entity should be displayed in **Balance History**

Reviewer: ```suggestion
Updated Balance entity should be displayed in **Balance History**.
```
